Testskript.R

```diff
@@ -8,4 +8,13 @@
 # Hallo Jule & Nicklas, das ist unser Testskript :-) Fuer mehr Smileys hier einmal ein: "JA"
 # :P
 
-# I want a dinosaur please! Preferably a tyrannosaurus
\ No newline at end of file
+# I want a dinosaur please! Preferably a tyrannosaurus
+
+Jule <- c(1, 2, 3, 4)
+Anne <- c(5, 6, 7, 8)
```

Comment:
These are example code lines, please put them below the function.

Reply:
Thanks for your feedback!